src/registers.rs

```diff
@@ -23,7 +23,7 @@ impl Registers {
     pub fn new() -> Self {
         Self {
             regs: [0; NUM_REGISTERS],
-            pc: PC_START,
+            pc: PC_START, // Program counter starts at 0x3000, lower addresses are reserved for OS
```

Comment:
Small nitpick, but is this the case?
I thought lover addresses were reserved for variables, not for the "OS"

Reply:
You are right Fabri, it is explained in the guide as following.
> The lower addresses are left empty to leave space for the trap routine code.